sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -326,6 +327,38 @@ public String getEntityPath() {
         return scheduleMessageInternal(message, scheduledEnqueueTime, null);
     }
 
+    /**
+     * Sends a scheduled messages to the Azure Service Bus entity this sender is connected to. A scheduled message is
+     * enqueued and made available to receivers only at the scheduled enqueue time.
+     *
+     * @param messages Message to be sent to the Service Bus Queue.
+     * @param scheduledEnqueueTime Instant at which the message should appear in the Service Bus queue or topic.
+     *
+     * @return The sequence number of the scheduled message which can be used to cancel the scheduling of the message.
+     *
+     * @throws NullPointerException if {@code messages} or {@code scheduledEnqueueTime} is {@code null}.
+     */
+    public Flux<Long> scheduleMessages(Iterable<ServiceBusMessage> messages, Instant scheduledEnqueueTime) {
```

Comment:
This should be OffsetDateTime or it will conflict with @YijunXieMS 's changes.

Reply:
I will wait for Yijun's change and merge it in my branch. Since it effect downstream methods.